README.md

```diff
@@ -18,23 +18,109 @@ FOLIO results for all supported (TBD) SIP2 commands.
 The edge-sip2 module can be launched via `edge-sip2-fat.jar` as follows:
 
 ```bash
-$ java -jar edge-sip2-fat.jar -conf '{"port":1234,"okapiUrl":"https://folio-snapshot-okapi.dev.folio.org","tenant":"diku"}'
+$ java -jar edge-sip2-fat.jar -conf sip2.conf
 ```
-On Windows the `edge-sip2-fat.jar` should be launched with the JSON configuration in double quotes and the inner double quotes should be escaped, for example:
+The -conf option can either specify the filename of the configuration or inline JSON. 
+Here is a sample sip2.conf file:
 ```
-$ java -jar edge-sip2-fat.jar -conf "{\"port\":1234,\"okapiUrl\":\"https://folio-snapshot-okapi.dev.folio.org\",\"tenant\":\"diku"\}"
+{ 
+  "port": 6443,
+  "okapiUrl": "https://folio-testing-okapi.dev.folio.org",
+  "tenantConfigRetrieverOptions": {
+    "scanPeriod": 300000,
+    "stores": [{
+      "type": "file",
+      "format": "json",
+      "config": {
+        "path": "sip2-tenants.conf"
+      },
+      "optional": false
+    }]
+  }
+}
+```
+For inline JSON, the format is:
+```
+-conf '{"port":1234,"okapiUrl":"https://folio-snapshot-okapi.dev.folio.org".....}'
+```
+On Windows, inline JSON configuration is in double quotes and the inner double quotes should be escaped, for example:
+```
+-conf "{\"port\":1234,\"okapiUrl\":\"https://folio-snapshot-okapi.dev.folio.org\"....}"
 ``` 
 
 |Config option|Type|Description|
 |-------------|----|-----------|
 |`port`|int|The port the module will use to bind, typically 1024 < port < 65,535.|
 |`okapiUrl`|string|The URL of the Okapi server used by FOLIO.|
-|`tenant`|string|The FOLIO assigned tenant ID. Multi-tenant support TBD.|
-|`fieldDelimiter`|string|The character that the self service kiosk will use when encoding SIP messages. Defaults to "\|".|
```

Comment:
Where do we define the field delimiter now? In the `scTenants` section?

Reply:
Oops, that got missed. I'll add it back into the scTenants section